docker/single-process/scripts/init

```diff
@@ -55,5 +55,9 @@ fi
 if [ -z $1 ]; then
   exec sudo -u huginn -H -E bundle exec unicorn -c config/unicorn.rb
 else
-  exec sudo -u huginn -H -E bundle exec rails runner "$@" RAILS_ENV=${RAILS_ENV}
+  if [[ $1 == sidekiq* ]]; then
+    exec sudo -u huginn -H -E bundle exec "$@" RAILS_ENV=${RAILS_ENV}
```

Comment:
Like [this](https://github.com/cantino/huginn/pull/1731/files#diff-8fb4cf3a04ba615729e3aa9ef015dd00R75), the problem was all the other background processes are run with `rails runner` which does not work for sidekiq.